mappings/net/minecraft/world/gen/feature/Feature.mapping

```diff
@@ -39,3 +39,14 @@ CLASS net/minecraft/class_3031 net/minecraft/world/gen/feature/Feature
 		ARG 0 posToState
 		ARG 1 pos
 		ARG 2 predicate
+	METHOD method_36997 (Lnet/minecraft/class_3494;Lnet/minecraft/class_2680;)Z
+		ARG 1 state
+	METHOD method_36998 predicatedSetBlockState (Lnet/minecraft/class_5281;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Ljava/util/function/Predicate;)V
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 state
+		ARG 4 predicate
+	METHOD method_36999 blockTagPredicate (Lnet/minecraft/class_2960;)Ljava/util/function/Predicate;
```

Comment:
```suggestion
	METHOD method_36999 notInBlockTagPredicate (Lnet/minecraft/class_2960;)Ljava/util/function/Predicate;
```
This returns a predicate that returns true if and only if the block state's block is not in a block tag with the given identifier.

My suggested name is quite bad; feel free to choose better names.

Reply:
Used your name, but keeping this conversation unresolved in case people come up with a better name